### ISSUE TITLE
No music directory? 

### ISSUE THREAD

**Issue by paulicka** [nisovin/Seven#1]
Shows up when you try to run... ;-)

**Comment by nisovin** [comment]
Yeah, I didn't want to put them in the repo. You can either comment out the music code (in main/R.gd), or you can download them from soundimage.org. They are in the chiptunes category.
